sdks/python/setup.py

```diff
@@ -478,7 +478,9 @@ def get_portability_package_data():
               # --extra-index-url or --index-url in requirements.txt in
               # Dataflow, which allows installing python packages from private
               # Python repositories in GAR.
-              'keyrings.google-artifactregistry-auth'
+              'keyrings.google-artifactregistry-auth',
+              # tox dependency which doesn't conflict with cachetools<6
+              'tox==4.27.0'
```

Comment:
"==" sounds strict. Should it be tox < 4.28.0 ?

Reply:
Fixed